src/components/Cloud/Bandwidth-Usage.js

```diff
@@ -13,10 +16,14 @@ import { convertBytesToGigabytes } from '../../utils';
  * @param {Array}   data    An array of usage data for the current site.
  */
 const BandwidthUsage = ( { loading, data } ) => {
+	if ( loading || ! Array.isArray( data ) ) {
```

Comment:
Ah I think the `withFetch` HoC default to sending an object. Either that or if the API response is empty and hasn't been cast to the appropriate type it be the issue. I'll have a think about some more improvements for that HoC to reduce the need for this kind of thing.

Reply:
Yeah, the problem I was trying to fix with that type check is one where the API request errors for some reason and returns a WP_Error, which ends up looking like an object when it gets to this point. I haven't really worked out an elegant way of handling it, but I think ideally the `withFetch` HOC should detect if the response matches the WP_Error shape, and put it in the `error` property instead of `data` if so. (That would also remove the need for the util methods I put in `utils/wp-error.js` which makes things cleaner in the final components.)